blueprints/ember-flexberry-account/index.js

```diff
@@ -0,0 +1,15 @@
+/* globals module */
+module.exports = {
+  afterInstall: function() {
+    var _this = this;
+    return _this.insertIntoFile(
```

Comment:
It seems we shoud alos include `<script src='https://www.google.com/recaptcha/api.js'></script>` on installing addon.

Reply:
https://github.com/Flexberry/ember-flexberry-account/blob/be64939b1e9a33972fe7ab2155faf00b053df63f/tests/dummy/app/index.html#L14